### DISCUSSION TITLE
help：OBS 30.0.0 ，plugin OBS-NDI 4.13.0，can not find my NDI HX camera，but NDI tools can find my NDI HX camera

### DISCUSSION THREAD

**Discussion by zhai-hello** [DistroAV/DistroAV#938] [Q&A - Support]
help：OBS 30.0.0 ，plugin OBS-NDI 4.13.0，can not find my NDI HX camera，but NDI tools can find my NDI HX camera
![Help OBS-NDI](https://github.com/obs-ndi/obs-ndi/assets/150640855/0b0989dd-c6b4-4abc-aec6-71baa21640a2)

**Comment by zhai-hello** [comment]
obs 28.0.0，plugin obs-ndi  4.10.0，using ndi-runtime-4.5.1-Windows，then it can work very well。maybe my computer's problem

**Reply by paulpv** [reply]
Would love to know why this happens.
I have not been able to figure it out.
Is it specifically an HX only problem?
Non-HX sources all work fine?

**Comment by Trouffman** [comment]
Tested with pixle 7 pro and OBS on win10 & 11

Make sure your OBS is allowed in the firewall (I assume you are on windows 10 or 11). 
Also check your network settings itself set to "Private"

please provide screenshot of your firewall settings

**Comment by zhai-hello** [comment]
I have solved this problem.Thank you.




仁慈de神
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "obs-ndi/obs-ndi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月9日(星期二) 晚上9:15
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [obs-ndi/obs-ndi] help：OBS 30.0.0 ，plugin OBS-NDI 4.13.0，can not find my NDI HX camera，but NDI tools can find my NDI HX camera (Discussion #938)





 
Tested with pixle 7 pro and OBS on win10 &amp; 11
 
Make sure your OBS is allowed in the firewall (I assume you are on windows 10 or 11).
 Also check your network settings itself set to "Private"
 
please provide screenshot of your firewall settings

**Reply by Trouffman** [reply]
Do you mind sharing what was the issue and how you solve it for other to know the solution ?

**Comment by zhai-hello** [comment]
reinstall Software




仁慈de神
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "obs-ndi/obs-ndi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年4月10日(星期三) 晚上8:09
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [obs-ndi/obs-ndi] help：OBS 30.0.0 ，plugin OBS-NDI 4.13.0，can not find my NDI HX camera，but NDI tools can find my NDI HX camera (Discussion #938)





 
Do you mind sharing what was the issue and how you solve it for other to know the solution ?
